src/main/java/seedu/duke/common/ErrorMessages.java

```diff
@@ -4,40 +4,48 @@
+            + "Please enter <help> for the command guide."),
+    ERROR_GLOBAL_INVALID_YEAR("Invalid year! (Note: Year should be 1000 - 9999) "
+            + "Please enter <help> for the command guide."),
+    ERROR_GLOBAL_INVALID_PERIOD("Type of period given is invalid, please enter <help> for the command guide."),
+    ERROR_GLOBAL_INVALID_NUMBER("Number for period stats given is invalid, "
+            + "please enter <help> for the command guide."),
+    ERROR_GLOBAL_NUMBER_NOT_NUMERIC("Non-integer detected, please enter <help> for the command guide."),
+    ERROR_GLOBAL_INDEX_NOT_NUMERIC("Non-numerical index detected, please enter <help> for the command guide."),
+    ERROR_TRANSACTION_INVALID_AMOUNT("Invalid amount! (Note: Amount must be positive integer, $10000000 or less only) "
+            + "Please enter <help> for the command guide."),
+    ERROR_TRANSACTION_INVALID_CATEGORY("Invalid category, please enter <help> for the command guide."),
+    ERROR_TRANSACTION_INVALID_DATE("Invalid date, please enter <help> for the command guide."),
+    ERROR_TRANSACTION_INVALID_TYPE("Type of transaction given is invalid, please enter <help> for the command guide."),
+    ERROR_FIND_COMMAND_MISSING_KEYWORDS("Keyword(s) for your search expression missing,"
+            + " please enter <help> for the command guide."),
+    ERROR_STATS_COMMAND_INVALID_TYPE("Type of statistics given is invalid, please enter <help> for the command guide."),
+    ERROR_STATS_INVALID_NUMBER("Invalid number! (Note: Stats number must between 1 - 100) "
+            + "Please enter <help> for the command guide."),
+    ERROR_UNKNOWN_HELP_OPTION("The parameter used for help option is unknown, "
+            + "please enter <help> for the command guide."),
     ERROR_STORAGE_TRANSACTION_CORRUPTED("Transaction values corrupted."
-            + " To preserve data, please STOP the program and edit your data file correctly."),
+            + " To PREVENT DATA OVERWRITE, please STOP the program and EDIT your duke.txt file correctly."),
     ERROR_STORAGE_BUDGET_CORRUPTED("Budget parameter is invalid or missing."
-            + " To preserve data, please STOP the program and edit your data file correctly."),
+            + " To PREVENT DATA OVERWRITE, please STOP the program and EDIT your duke.txt file correctly."),
     ERROR_STORAGE_WRITE("Unable to write to Duke.txt. Please save your current Duke.txt file and restart Moolah"),
     ERROR_MAXIMUM_TRANSACTION_COUNT_REACHED("Unable to add transaction. "
             + "The maximum allowed transaction size (1000000) has been reached."),
 
-    ERROR_INVALID_BUDGET("Invalid budget amount, please ensure your amount is in positive whole number"
-            + " of valid range only!"),
+    ERROR_INVALID_BUDGET("Invalid budget amount! (Note: Budget must be a positive integer of valid range)"
+            + " Please enter <help> for the command guide."),
```

Comment:
Usage of integer here is not very accurate since from the programming point of view, the budget is a long instead of integer. Perhaps using whole number here will be better for the budget.